upgrade_test.py

```diff
@@ -666,12 +668,15 @@ def test_generic_cluster_upgrade(self):  # pylint: disable=too-many-locals,too-m
         we want to use this case to verify the read (cl=ALL) workload works
         well, upgrade all nodes to new version in the end.
         """
-
+        ignore_error_msgs = []
+        step_idx = 0
         # prepare workload (stress_before_upgrade)
         InfoEvent('Starting stress_before_upgrade - aka prepare step')
         stress_before_upgrade = self.params.get('stress_before_upgrade')
         prepare_thread_pool = self.run_stress_thread(stress_cmd=stress_before_upgrade)
-
+        if self.params.get('alternator_port'):
+            self.pre_create_alternator_tables()
+            ignore_error_msgs.append((YcsbStressEvent, '.*Unable to execute HTTP request: Connection refused.*'))
```

Comment:
why are you creating a list for only one item ?

Reply:
so it can be empty...
I don't like doing a whole bunch of code with no-op like that...

I'm o.k. with filtering all the time this error, since it's very specific, and it won't be there in CQL only case.

Reply:
Done